### DISCUSSION TITLE
Adding beatmap to a collection shouldn't reset the dropdown

### DISCUSSION THREAD

**Discussion by ricterG** [ppy/osu#20961] [Ideas]
If you have more than a few collections, it will show them in the dropdown with a scrollbar. If you scroll to the bottom and then add or remove a beatmap to one of the collections, it will then reset and immediately scroll to the top of the list again. I don't think this is ideal behavior, it makes it annoying to add a beatmap to multiple collections that are at the bottom of my list, since I have to scroll down multiple times. Not sure if this is technically a bug or not

[collections.webm](https://user-images.githubusercontent.com/41763965/198174337-40f862dc-b831-47f0-bf7d-299abd3035fc.webm)

**Comment by smoogipoo** [comment]
This must be a regression, I don't remember it doing this. But it's likely also going to get a redesign so not sure how important it is for the time being.
